index.js

```diff
@@ -64,7 +64,7 @@ module.exports = class Tailor extends EventEmitter {
 
         requestOptions.parseTemplate = parseTemplate(
             [requestOptions.fragmentTag].concat(requestOptions.handledTags),
-            ['script', requestOptions.fragmentTag]
+            ['script', requestOptions.fragmentTag].concat(requestOptions.handledTags)
```

Comment:
@vigneshshanmugam I guess you were meant this. Those custom `handledTags` where not passed to pipe tags, so for my `switcher` example it was not added. Not sure it's the best way, though.

Reply:
Yeah you are right. Just feels like this gives too many options of having where pipe should be inserted. Actually, pipe is never required if its not a fragment, "script" was added because anyone can add fragment in the head like `<script src="blah" type="fragment"/>`.

I would move the `fragment` in your example before `switcher` to make things easier.

Reply:
Or have a script tag :)
It was an interesting side-effect I didn't think about.
People who implement their own tags should kinda be aware then :)

Reply:
Yeah true, lets remove these options and insert a empty div tag like before.